src/pages/designing/figma-kit.mdx

```diff
@@ -5,22 +5,162 @@ description: The new Figma design kit contains type styles, color tokens, and co
+<AnchorLink>Using the Figma kit</AnchorLink>
+<AnchorLink>Other Figma resources</AnchorLink>
+
+</AnchorLinks>
+
+## Accessing the new Figma kit
 
 <Row>
 <Column  colLg={8} colMd={8} colSm={4}>
 
-The Carbon for IBM.com team is alpha testing a Figma kit that contains type
-styles and all color tokens in the one kit.
+We have a new Figma kit for delivering pre-built versions of the Carbon for IBM.com components. The kit contains libraries for each theme and includes color tokens, expressive and productive type styles, components, and grid templates.
+
+We’ll update the kit with more components and adjustment to existing components over time, and Figma will notify you when library updates are available.
+
+If you are in the IBM organization in Figma, you will automatically have access to the Carbon for IBM.com Figma libraries.
+
+### Activating the new Figma kit
+
+If you are in the IBM Figma organization, you can access the Carbon for IBM.com Figma kits via your Figma file.
+
+#### To use one or all of the themes in your Figma file:
+
+1. In the menu, click the Figma dropdown then select `Libraries`.
+2. In the Libraries list, search for `Carbon for IBM.com`.
+3. Select the theme to enable.
+
+![Activating the Figma libraries](../../images/get-started/Designing_Libraries_Figma.gif)
+
+<Caption>
+  The Carbon for IBM.com themes are available from the Libraries list.
+</Caption>
+
+### Figma tutorials
+
+If you are new to Figma, they offer <a href="https://www.youtube.com/c/Figmadesign/playlists?view=50&sort=dd&shelf_id=3" target="_blank" rel="noopener noreferrer">video tutorials</a> on their official YouTube channel.
+
+## Using the Figma kit
+
+Our Figma resources are set up to help you design using key IBM assets. Learn more about using the [2x Grid](#2x-grid), [Components](#components), [Text styles](#text-styles), and [Color styles](#color-styles) in Figma.
+
+### 2x Grid
+
+The IBM 2x Grid is the framework for laying out all visual elements.
+
+#### Understanding the grid
+
+For a detailed walkthrough of the 2x Grid, see the foundational 2x Grid guidance on the Carbon Design System site. The [Overview](https://www.carbondesignsystem.com/guidelines/2x-grid/overview) provides the background and design rationale for the 2x Grid. [Implementation](https://www.carbondesignsystem.com/guidelines/2x-grid/implementation) will get you up and running, designing with the grid.
+
+For information about using the 2x Grid with web pages, including best practices, see our [Layout guidelines](../guidelines/layout).
+
+#### Start using the grid
+
+In Figma, frames function as artboards and containers for everything from screens to buttons. You can learn more about frames in Figma’s <a href="https://help.figma.com/hc/en-us/articles/360041539473-Frames-in-Figma" target="_blank" rel="noopener noreferrer">documentation</a>. You can apply the grid template to a frame in Figma if you have one or more of the Carbon for IBM.com libraries active:
```

Comment:
> You can apply the grid template to a frame in Figma if you have one or more of the Carbon for IBM.com libraries active:

Suggest change to: 

> You can apply the grid template to a frame by following these steps:

Reply:
Tweaking this language slightly - I realized there were two sentences with colons introducing the same thing: 

> You can apply the grid template to a frame by following the steps below.